.eslintignore

```diff
@@ -1 +1,2 @@
-!.storyboook
+node_modules/
+docs/
```

Comment:
Needed so eslint doesn't look at these folders.

(And we do want the .storybook/ files to be linted.)